ui_framework/components/_overrides.scss

```diff
@@ -0,0 +1,7 @@
+.fullWidth {
```

Comment:
Normally this kind of file ends up being called "utility" in frameworks. I'm guessing we'll add a lot more stuff to it over time (especially as we add more strict spacing guidelines). We might want to prefix it as such so that people understand they are their own things not tied to the classes they overwrite. `.kuiUtility--thing`